doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1728,14 +1754,6 @@ Declaring Implicit Arguments
    To know which are the implicit arguments of an object, use the
    command :cmd:`Print Implicit` (see :ref:`displaying-implicit-args`).
 
-.. warn:: Argument number @num is a trailing implicit so must be maximal.
```

Comment:
Documentation of this warning should not be removed, but should instead be changed to be documenting an error.

Reply:
This is the case but I moved it, see  lines 1586 to 1592:
```
.. exn:: Argument @ident is a trailing implicit, so it can't be declared non maximal. Please use %{ %} instead of [ ].

   For instance:

   .. coqtop:: all fail

      Fail Definition double [n] := n + n.
```